addons/mail/static/src/models/composer/composer.js

```diff
@@ -1110,6 +1242,11 @@ function factory(dependencies) {
         thread: one2one('mail.thread', {
             inverse: 'composer',
         }),
+        // This will containe the sending message queue
+        queue: one2many('mail.message'),
+        queueProcess: attr({
+            default: false
+        }),
```

Comment:
name not explicit enough (at least missing the goal of the queue: to post messages), and missing/bad docstring format

so instead of queue, name suggestion: messagesPendingPost, and instead of queueProcess, hasMessagePostInProgress / isPostingMessage